src/In.ProjectEKA.DefaultHip/Resources/dignosticReportWithPresentedFormPdfFileAsUrl.json

```diff
@@ -0,0 +1,35 @@
+{
+  "resourceType": "Bundle",
+  "id": "11123232324",
+  "type": "collection",
+  "entry": [{
+    "fullUrl": "12387re4337483",
```

Comment:
same as above comment on the fullURL. use "urn:uuid:<uuid>" format. 
resource "id" name should be "uuid" that you have given above as part of the fullUrl